src/css/betterttv-dark.css

```diff
@@ -58,13 +58,18 @@ body {
     background: #111 !important;
 }
 
+/* brighten default dark star sub badge */
+.ember-chat .badges img.badge[src="https://static-cdn.jtvnw.net/badges/v1/19dd8673-124d-4f44-830c-b0f4f9d78635/1"] {
```

Comment:
That brightens every subscriber badge including custom ones uploaded by partnered streamers